htp/htp_config_private.h

```diff
@@ -238,6 +238,12 @@ struct htp_cfg_t {
     htp_hook_t *hook_request_body_data;
 
     /**
+     * Receives raw request body data, starting immediatelly after empty line
+     * that terminates list of request headers.
+     */
+    htp_hook_t *hook_request_raw_body_data;
```

Comment:
This is not user-accessible structure. Should I fix that? So, if user upgrades libhtp, binary compatibility issue shid should never happen. does not it ?